Donation-platform/src/main/webapp/personalizationScript.js

```diff
@@ -47,52 +47,47 @@ function getPersonalizedCharitiesFromServlet() {
 }
 
 /* (2) Displays the list of personalized charities them on personalized.html */
-function updatePersonalizedList(charities) {
-  const personalizedListElement = document.getElementById('personalized-list');
-
-  // Remove old personalized results
-  personalizedListElement.innerHTML = "";
-  
-  // Displays charity listings in the format: "[Charity Name] ([tag 1], [tag2], ...)"
-  charities.forEach(charity => {
-    var charityString = "<li><strong>" + charity.name + "</strong>  (";
-    for(i = 0; i < (charity.tags).length; i++) {
-      if(i == (charity.tags).length - 1) {
-        charityString += (charity.tags)[i] + ")</li>";
-      } else {
-        charityString += (charity.tags)[i] + ", ";
-      }
-    }
-    personalizedListElement.innerHTML += charityString;
-  });
-}
-
-/**
- * Updates the display with bootstrap cards.
- */
 function updatePersonalizedCardsOnPage(charities) {
-    const cards = document.getElementById('cards');
+    const cards = document.getElementById('cont');
```

Comment:
what is "cont" stands for? If possible please rename

Reply:
Done.